### PR TITLE
[Access for SaaS] Create Greenhouse Recruiting tutorial

### PR DESCRIPTION
**Pull request by chcyrier**
PCX-11296

### CODE REVIEW

**Review by kennyj42 — APPROVED** [review]

File: content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+weight: 8
+---
+
+# Connect to Greenhouse Recruiting through Access
+
+This guide covers how to configure [Greenhouse Recruiting](https://support.greenhouse.io/hc/en-us/articles/360040753811-Configure-single-sign-on-SSO-for-Greenhouse-Recruiting) as a SAML application in Cloudflare Zero Trust.
+
+## Prerequisites
+
+- A [SAML identity provider](/cloudflare-one/identity/idp-integration/generic-saml/) configured in Cloudflare Zero Trust
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md:13]:
```suggestion
- A [SAML identity provider](/cloudflare-one/identity/idp-integration/generic-saml/) configured in Cloudflare Zero Trust.
```

File: content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- Admin access to an Advanced or Expert Greenhouse Recruiting site
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md:14]:
```suggestion
- Admin access to an Advanced or Expert Greenhouse Recruiting site.
```

File: content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## 1. Add a SaaS application to Cloudflare Zero Trust
+
+1. In [Zero Trust](https://one.dash.cloudflare.com), go to **Access** > **Applications**.
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md:18]:
```suggestion

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).
2. Select **Zero Trust**.
3. Go to **Access** > **Applications**.
```

Comment (Maddy-Cloudflare) [reply content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md:18]: The numbered list would need to be reordered, as I added two steps.

File: content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## 3. Add a SAML SSO provider to Greenhouse
+
+1. In Greenhouse Recruiting, go to the **Configure** icon > **Dev Center** > **Single sign-on**.
+2. Copy the **SSO Assertion Consumer URL**.
+3. Under **Upload XML file**, select **Choose a file**, and upload the `.xml` file created in step [2. Download the metadata file](#2-download-the-metadata-file).
+4. Change the **Entity ID** to `greenhouse.io`.
+5. Keep this window open without selecting **Begin testing**. You will finish this configuration in step [5. Test the integration and finalize configuration](#5-test-the-integration-and-finalize-configuration).
+
+## 4. Finish adding a SaaS application to Cloudflare Zero Trust
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md:39]:
```suggestion
## 4. Finish adding a SaaS application to Cloudflare Zero Trust

```

File: content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+1. In your open Zero Trust window, fill in the following fields:
+    - **Entity ID**: `greenhouse.io`
+    - **Assertion Consumer Service URL**: SSO Assertion Consumer URL from SSO configuration in Greenhouse Recruiting.
+    - **Name ID format**: _Email_
+2. Select **Save configuration**.
+3. Configure [Access policies](/cloudflare-one/policies/access/) for the application.
+4. Select **Done**.
+
+## 5. Test the integration and finalize configuration
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-one/applications/configure-apps/saas-apps/greenhouse-saas.md:49]:
```suggestion
## 5. Test the integration and finalize configuration

```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**